server/channels/api4/license.go

```diff
@@ -93,7 +95,10 @@ func addLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 	defer file.Close()
 
 	buf := bytes.NewBuffer(nil)
-	io.Copy(buf, file)
+	if _, err := io.Copy(buf, file); err != nil {
+		c.Err = model.NewAppError("addLicense", "api.license.add_license.copy.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
```

Comment:
This is a new error ID right? Please run `make i18n-extract` to update the translation file and add the error text.

Reply:
Done already. Thank you for the review.